cpt/test/integration/docker_test.py

```diff
@@ -163,3 +162,36 @@ def build(self):
             self.packager.run()
             self.assertIn("--cpus=1  conanio/gcc8", self.output)
             self.assertIn("/home/conan/project:z", self.output)
+
+    @unittest.skipUnless(is_linux_and_have_docker(), "Requires Linux and Docker")
+    def test_docker_run_android(self):
+        self.create_project()
+        command = ('docker run --rm -v "{}:/home/conan/project" ',
```

Comment:
It's not possible to read stdout from run_in_docker when there is Docker container running.